python/dgl/convert.py

```diff
@@ -1043,6 +1045,56 @@ def to_homogeneous(
         return retg
 
 
+def bipartite_from_sparse(
+    sp_mat: SparseMatrix,
+    utype: str = "_U",
+    etype: str = "_E",
+    vtype: str = "_V",
+    eweight_name: str = None,
+):
```

Comment:
Not sure if this is indeed needed as users can always create a DGLGraph by the rows and columns of a SparseMatrix. Note that there are also other options such as extending the `data` argument of `dgl.graph` and `dgl.heterograph` to allow SparseMatrix type. But if we do believe this is useful, let's create another PR for this new API and add proper tests/docs for it.

Reply:
OK. I removed this interface.